v21.2/authorization.md

```diff
@@ -145,6 +145,11 @@ We recommend the following best practices to set up access control for your clus
 - Use the `root` user only for database administration tasks such as creating and managing other [users](#sql-users), creating and managing [roles](#roles), and creating and managing databases. Do not use the `root` user for applications; instead, create users or roles with specific [privileges](#assign-privileges) based on your application’s access requirements.
 - Use the ["least privilege model"](https://en.wikipedia.org/wiki/Principle_of_least_privilege) to grant privileges to users and roles.
 
+We recommend the following best practices for optimal performance in `ROLE` operations ([`CREATE ROLE`](create-role.html), [`ALTER ROLE`](alter-role.html), [`DROP ROLE`](drop-role.html)):
```

Comment:
I do think a bit of the "why" would be helpful. it's not really a performance concern with the ROLE operations themselves -- so we should say that doing it this way will have less of an effect on the latency of logging into a new session.

otherwise people may see this advice, think to themselves, "i don't have any performance issues with this command," and then ignore the advice.

Reply:
Yeah, you're totally right. I updated the PR with some more details, without going into too much detail. PTAL.